src/scenes/Event/components/ResultHistory/index.js

```diff
@@ -76,6 +76,7 @@ export default class EventResultHistory extends Component {
             <TableBody>
               {_.map(sortedOracles, (oracle, index) => {
                 const invalidOption = localizeInvalidOption(oracle.options[oracle.resultIdx], intl);
+                if (currentEvent.phase === Phases.VOTING && index === sortedOracles.length - 1) return;
```

Comment:
Lets remove the Oracle object from `sortedOracles` for Voting phase instead of not returning the the `TableRow`

Reply:
Tried simply filter Voting phase at sortedOracles but failed. It will affect other oracle pages because of the magic codes here.